app/scripts/utils/get-default-track-for-datatype.js

```diff
@@ -0,0 +1,33 @@
+
+// Configs
+import {
+  DEFAULT_TRACKS_FOR_DATATYPE,
+} from '../configs';
+
+/**
+ * Gets the default track as defined in utils/default-tracks-for-datatype.js
+ *
+ * @param  {string} datatype The datatype to get the default track for
+ * @param  {string} position   top, bottom, left, right, center
+ * @param  {array} availableTracks   list of tracks to choose from,
+ * typically obtained from AVAILABLE_TRACK_TYPES(...)
+ * @return {object}  an element of availableTracks or undefined
+ */
+const getDefaultTrackForDatatype = (datatype, position, availableTracks) => {
+  if (availableTracks.length === 0) { return undefined; }
+
+  if (availableTracks.length === 1) { return availableTracks[0]; }
+
+  let usedTrackType = availableTracks[0];
+  const defaultTrackType = DEFAULT_TRACKS_FOR_DATATYPE[datatype] !== undefined
+    ? DEFAULT_TRACKS_FOR_DATATYPE[datatype][position]
+    : undefined;
+
+  if (defaultTrackType !== undefined) {
+    usedTrackType = availableTracks.find(tt => tt.type === defaultTrackType) || usedTrackType;
```

Comment:
Would you mind adding a short comment explaining why this is necessary?

Something like "We found a default track type for this datatype, now we need to find the definition of this track."